query-engine/js-connectors/js/neon-js-connector/tsconfig.build.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "declaration"
```

Comment:
The idea would be that the output of `pnpm lint` goes in `./declaration` to avoid writing conflicts with `tsup`'s output, which goes into `./dist`, the actual `outDir` for the main `tsconfig.json` file. Open to change this later on in whatever way makes this less confusing. But it's also a very minor thing, one could probably remove the `lint` command altogether for these packages